documentation/automatic-bisection.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: documentation
+title: Automatic bisection
+doc_order: 10
+description: How to evaluate tested build automatically with a command line
+---
+
+# Automatic bisection
+
+Mozregression allows to evaluate builds automatically with a command line
+instead of interactively asking you.
```

Comment:
"allows to" isn't the best grammar, I'd probably say something like:

"As an alternative to using mozregression's interactive mode, it is possible to evaluate builds automatically by writing a custom script. This can save a lot of time if the reproduction steps are complicated or you have a large regression range."